README.rst

```diff
@@ -0,0 +1,139 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME - include uses, unique features
+PDFmorph is a Python software package designed to increase the insight
+researchers can get into spectra collected for atomic pair distribution 
+functions.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
+run. If you are unfamiliar with the CLI, consult online resources on how to get 
+started with it on your machine.
+
+It is recommended that you use Anaconda Python to run this application; to do so,
+first make sure that you have installed it from Anaconda distribution here:
+https://www.anaconda.com/distribution/.
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+To ensure the installation worked, activate the relevant conda environment and 
+type ::
+
+	pdfmorph --version
+
+If installed correctly, this command should return PDFmorph's current version
+number.
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+#FIXME - easy_install?
```

Comment:
@CJ-Wright @sbillinge Some sources on PyPl and stuff I've seen recently suggest `pip` over `easy_install` - does our infrastructure work with that/would it be worth it to encourage `pip` over `easy_install?`

Reply:
I am easy either way.  @pavoljuhas and @CJ-Wright  may both want to chime in.

Reply:
`easy_install` and `pip` have conflicting way of handling namespaced packages (`diffpy` functions as a namespace only and is not a real package) and a mixed installation then may not work:

```sh
$ conda create -n xx36 -c diffpy diffpy.utils
$ conda activate xx36
$ python -c 'import diffpy.utils'    # OK
$ pip install https://github.com/diffpy/diffpy.pdfmorph/archive/master.tar.gz
$ python -c 'import diffpy.utils'    # FAILS
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'diffpy.utils'
```

This actually happens because of an outdated definition of namespace package in [diffpy/__init__.py](
https://github.com/diffpy/diffpy.pdfmorph/blob/2e3f047d01b501641e41b6ba65e43fde1886b274/diffpy/__init__.py#L21) which is specific to setuptools (`easy_install`) and does not work well with pip.  Other diffpy packages have that code updated, e.g., [diffpy.structure/src/diffpy/__init__.py](https://github.com/diffpy/diffpy.structure/blob/df264a84139bc834424cc323d1bcfc1f9a4ca6cc/src/diffpy/__init__.py#L22-L23), and then it seems to work with both `easy_install` and `pip`.

The bottom line - for now I propose to:
1. within this PR leave `easy_install` recommendation in the README as is
2. create a new issue to
   - fix namespace package handling in `diffpy/__init__.py` as in diffpy.structure and
   - check if mixed `easy_install`, `pip` installation of several diffpy packages works OK
   - and if it does, update README to suggest `pip` as the installation tool